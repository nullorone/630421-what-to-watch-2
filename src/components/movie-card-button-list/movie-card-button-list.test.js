import React from "react";
import renderer from "react-test-renderer";
import MovieCardButtonList from "./movie-card-button-list";

describe(`Make snapshot`, () => {
  const initProps = {
    buttons: [{
      iconName: `pepper`,
      classModifier: `inactive`,
      text: `Red Hot Chili Peppers`,
    }]
  };

  it(`Get snapshot component`, () => {
    const tree = renderer
      .create(
          <MovieCardButtonList {...initProps}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
