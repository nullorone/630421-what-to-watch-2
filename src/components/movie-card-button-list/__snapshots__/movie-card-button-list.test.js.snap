// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make snapshot Get snapshot component 1`] = `
<div
  className="movie-card__buttons"
>
  <button
    className="btn btn--inactive movie-card__button"
    type="button"
  >
    <svg
      height="19"
      viewBox="0 0 19 19"
      width="19"
    >
      <use
        xlinkHref="#pepper"
      />
    </svg>
    <span>
      Red Hot Chili Peppers
    </span>
  </button>
</div>
`;
